src/test/java/org/opentripplanner/transit/raptor/moduletests/A04_NoWaitTest.java

```diff
@@ -0,0 +1,132 @@
+package org.opentripplanner.transit.raptor.moduletests;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.opentripplanner.transit.raptor._data.transit.TestRoute.route;
+import static org.opentripplanner.transit.raptor._data.transit.TestTransfer.walk;
+import static org.opentripplanner.transit.raptor._data.transit.TestTripSchedule.schedule;
+import static org.opentripplanner.transit.raptor.api.request.RaptorProfile.MULTI_CRITERIA;
+import static org.opentripplanner.transit.raptor.api.request.RaptorProfile.NO_WAIT_STD;
+import static org.opentripplanner.transit.raptor.api.request.RaptorProfile.STANDARD;
+import static org.opentripplanner.transit.raptor.api.request.SearchDirection.REVERSE;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.transit.raptor.RaptorService;
+import org.opentripplanner.transit.raptor._data.RaptorTestConstants;
+import org.opentripplanner.transit.raptor._data.api.PathUtils;
+import org.opentripplanner.transit.raptor._data.transit.TestTransitData;
+import org.opentripplanner.transit.raptor._data.transit.TestTripSchedule;
+import org.opentripplanner.transit.raptor.api.request.RaptorRequestBuilder;
+import org.opentripplanner.transit.raptor.rangeraptor.configure.RaptorConfig;
+
+/**
+ * FEATURE UNDER TEST
+ *
+ * This test focus on testing access and egress with basic Raptor - NOT Range Raptor. All tests are
+ * run with one iteration only. The Range Raptor make sure the best access/egress is used, but
+ * without it, we should board at the first possible stop for the standard raptor, but not for the
+ * no-wait-std. The reason for this is that the standard optimize on best arrival time, while the
+ * no-wait-std optimize on finding the shortest travel time.
```

Comment:
I have trouble understanding the "The Range Raptor make sure the best access/egress is used, but without it, we should board at the first possible stop for the standard raptor, but not for the no-wait-std" sentence but I guess "but without it" refers to basic Raptor? Also do basic and standard Raptor refer to different things (basic = not Range Raptor, standard = not multicriteria)?

Reply:
Good points. There has been a bit of a evolution here, so maybe a bit of renaming is in its place:
**Raptor Strategies**
  - `StdTransitWorker` rename to `ArrivalTimeRaptorStrategy`
  - `NoWaitTransitWorker` rename to `MinTravelDurationRaptorStrategy`
  - `McTransitWorker` rename to `MultiCriteriaRaptorStrategy`
 
 There are also a _Standard_ state which can be combined with the 2 first strategies above - witch I guess make it even more confusing. The origin of the "Standard" refer to the first original paper, but it is too vague - so we should change it. There is a bit of documentation and code to update here, so I would like to do it in a separate PR. I can create an issue for this if you all agree that this is a good enhancement. All of this should be documented in the https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/src/main/java/org/opentripplanner/transit/raptor/package.md.

Reply:
Sounds like a good plan to me.